pkg/promtail/targets/filetarget_test.go

```diff
@@ -27,8 +29,7 @@ func TestLongSyncDelayStillSavesCorrectPosition(t *testing.T) {
 
 	err := os.MkdirAll(dirName, 0750)
 	if err != nil {
-		t.Error(err)
-		return
+		t.Fatal(err)
```

Comment:
Should have mentioned this earlier, but I prefer `require.NoError(t, err)` instead of these 3 lines.  We can come back and tidy this up.